src/Tribe/Service_Providers/First_Boot.php

```diff
@@ -60,6 +60,16 @@ public function set_default_options() {
 			$options['tribeEventsTemplate'] = '';
 		}
 
+		if ( ! isset( $options['tribeEnableViews'] ) ) {
+			// Set the Enabled Views to List, Month, and Day
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 2132490d3641337e3e505e58eaa53e72</sub>